pyccel/codegen/printing/fcode.py

```diff
@@ -452,28 +456,112 @@ def _print_Import(self, expr):
         return self._get_statement(code) + '\n'
 
     def _print_PythonPrint(self, expr):
+        end = '\n'
+        sep = ' '
+        code = ''
+        empty_end = FunctionCallArgument(LiteralString(''), 'end')
+        space_end = FunctionCallArgument(LiteralString(' '), 'end')
+        kwargs = [f for f in expr.expr if f.has_keyword]
+        for f in kwargs:
+            if f.keyword == 'sep':
+                sep = str(f.value)
+            elif f.keyword == 'end':
+                end = str(f.value)
```

Comment:
I noticed that `kwargs` is used only in the forloop.
it may be better to use only
```python
for f in expr.expr:
     if not f.has_keyword:     continue
     if f.keyword == 'sep':    sep = str(f.value)
     elif f.keyword == 'end':  end = str(f.value)
```
without creating kwargs list only for this forloop

Reply:
@rakati i tried to keep the same structure as the one used in ccode.py. shall i change both of them?

Reply:
you may not need, because Fortran and C are different, so there may be an easy way to do it instead of using the same structure as in `ccode`.